app/multitenant/dynamo_collector.go

```diff
@@ -95,14 +95,19 @@ type DynamoDBCollector interface {
 	CreateTables() error
 }
 
+type ReportStore interface {
+	FetchReports([]string) ([]report.Report, []string, error)
+}
```

Comment:
Just out of interest, why did you go for a full blow interface and not just a function type?